config/config.go

```diff
@@ -722,14 +722,6 @@ func (self *CGRConfig) checkConfigSanity() error {
 				utils.DispatcherS, self.dispatcherSCfg.DispatchingStrategy)
 		}
 	}
```

Comment:
please keep the check here, just only  "for _, connCfg := range self.schedulerCfg.CDRsConns", without self.schedulerCfg.Enabled